testsuite/cluster-test/src/tx_emitter.rs

```diff
@@ -389,12 +387,16 @@ impl SubmissionWorker {
 async fn wait_for_accounts_sequence(
     client: &mut AdmissionControlClientAsync,
     accounts: &mut [AccountData],
+    instance: &Instance,
```

Comment:
You can consider replacing `client` with`NamedAdmissionControlClient` here,  instead of adding extra param

Reply:
Done